src/Phinx/Migration/AbstractMigration.php

```diff
@@ -44,6 +44,12 @@
  */
 abstract class AbstractMigration implements MigrationInterface
 {
+    const PHINX_PARAM_BOOL = \PDO::PARAM_BOOL;
+    const PHINX_PARAM_NULL = \PDO::PARAM_NULL;
+    const PHINX_PARAM_INT = \PDO::PARAM_INT;
+    const PHINX_PARAM_STR = \PDO::PARAM_STR;
+    const PHINX_PARAM_LOB = \PDO::PARAM_LOB;
```

Comment:
This was a design from #256.

I believe the goal is to abstract away the adapter that is being used in the migrations your write.

It's not something I would have thought to do myself, but I believe the author, @cyrusboadway, was right to do it this way.

There is also an abstract `getAdapterBindParamType()` method now such that if an alternative to PDO is introduced, this method will accept one these constants and return a value suitable for the new adapter.

Phinx is clearly designed to accommodate different adapters, so I think it makes sense to provide constants that are not specific to one adapter (even if the constants values are based on one in particular).